test/simple_socketio.test.js

```diff
@@ -67,6 +69,9 @@ describe("Tests using SimpleSocketIOClient", () => {
         credentials.apiKey,
       );
 
+      await instance1.initializing;
+      await instance2.initializing;
```

Comment:
this fixes unhandled network requests once a test has finished, which it complained about